src/request_template.rs

```diff
@@ -314,10 +368,14 @@ mod tests {
     let tmpl = RequestTemplate::try_from(endpoint).unwrap();
     let ctx = Context::default();
     let req = tmpl.to_request(&ctx).unwrap();
+    println!("testing post");
     assert_eq!(req.method(), reqwest::Method::POST);
+    println!("testing headers");
     assert_eq!(req.headers().get("foo").unwrap(), "bar");
     let body = req.body().unwrap().as_bytes().unwrap().to_owned();
+    println!("testing body");
     assert_eq!(body, "foo".as_bytes());
+    println!("testing url");
```

Comment:
drop all `println`

Reply:
@tusharmath  I've removed the println statements